src/quilt/poetry.lock

```diff
@@ -0,0 +1,20 @@
+[[package]]
+name = "portablemc"
+version = "3.1.1"
+description = "PortableMC is a module that provides both an API for development of your custom launcher and an executable script to run PortableMC CLI."
+category = "main"
+optional = false
+python-versions = ">=3.6"
+develop = true
+
+[package.source]
+type = "directory"
+url = "../core"
+
+[metadata]
+lock-version = "1.1"
+python-versions = ">=3.6"
+content-hash = "53ec67e3c6a3090dcb122ad5f2d8d43c31b3745d0aa5e7362c0b02ef26e620f3"
+
+[metadata.files]
+portablemc = []
```

Comment:
The lock file seems out-of-place, I think this requires re-running poetry lock in the quilt package

Reply:
I re-runned poetry lock but unchanged

Reply:
Ahhh my bad I wasn't reading it correctly!